specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc

```diff
@@ -1146,25 +1143,25 @@ Specifications that define their own additional deployment descriptors
 should provide a _metadata-complete_ attribute of their own, if deemed
 useful, with the appropriate semantics.
 
-=== Deploying a Stand-Alone Java EE Module
+=== Deploying a Stand-Alone Jakarta EE Module
 
 This section specifies the requirements for
-deploying a stand-alone Java EE module.
+deploying a stand-alone Jakarta EE module.
 
-. The deployment tool must first read the Java
+. The deployment tool must first read the Jakarta
 EE module deployment descriptor if provided externally to the package or
 if present in the package. See the component specifications for the
 required location and name of the deployment descriptor for each
 component type.
 . If the deployment descriptor is absent, or
-is present and is a Java EE 5 or later version descriptor and the
+is present and is a Jakarta EE 5 or later version descriptor and the
```

Comment:
not sure whether this should say Java EE 5 as there is no jakarta EE 5

Reply:
agreed. reverted